sdk/maps/maps-route-rest/src/helpers.ts

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { LatLon } from "@azure/maps-common";
+import { BatchRequest, RouteGetRouteDirectionsQueryParamProperties } from "./generated";
+
+function toLatLonString(coordinates: LatLon): string {
+  return `${coordinates[0]},${coordinates[1]}`;
+}
+
+/**
+ * Transform an array of [Latitude, Longtitute] to a string in the following format:
+ * "Latitude_1,Longtitute_1:Latitude_2,Longtitute_2:..."
+ *
+ * @param coordinates - An array of Latitude/Longtitute pair to transform.
+ * @returns The transformed string.
+ */
+export function toColonDelimitedLatLonString(coordinates: LatLon[]): string {
+  return coordinates.map((c) => toLatLonString(c)).join(":");
+}
+
+const clientToServiceNames: Readonly<Record<string, string>> = {
```

Comment:
Why do we need this mapping? We usually try to keep service names in Rest Clients

Reply:
I try to rename several properties to align the name with the high-level client version. If we shouldn't do any name transform, let me remove the directives

Reply:
Per our previous conversation, we are now delivering the server name. If users request renaming, we will provide a helper function.
Changed in 0e52644759f2799edf296f2f8f4a7b8695721b85